src/olympia/discovery/data.py

```diff
@@ -82,3 +82,66 @@ def __repr__(self):
     DiscoItem(
         addon_id=21085, type=amo.ADDON_PERSONA, addon_name='Evil Robots'),
 ]
+
+"""
+China Edition Firefox shows a different selection of add-ons.
+See discopane_items comments above for more detail on format.
+"""
+discopane_items_china = [
+    # 'Vintage Fabric' theme.
```

Comment:
I wonder if this should be a dict that can fit various editions, e.g

```python
discopane_items = {
    'china': [...],
    'default': [...]
}
```

'cause it seems that we disable TAAR as soon as we have a separate edition? Not sure but if that's the case we could handle it like that.

Reply:
we could have discopane_items like that, sure.  I don't see how refactoring it would affect TAAR support or not though.

Reply:
well, nevermind with the TAAR thing. It'd be just, the most simplest version is that the view changes TAAR support based on whether there is an `edition` in the params but doesn't change it's behavior based on what edition it is. But it's easy to add later if we need that :)

Reply:
tbh, I've no idea if there will be any further special edition support, and if additional `edition=xxx` will have TAAR support or not.  `edition` was my attempt at making it a little future proof, as an alternate to having something like `?chinaedition=true` in the url which starts to get messy quickly.

Reply:
https://github.com/mozilla/addons-server/pull/7493/commits/45f01a497264cfe3d173625d7e70d64a5d0b9334